lms/djangoapps/student_account/views.py

```diff
@@ -592,3 +596,45 @@ def account_settings_context(request):
         } for state in auth_states if state.provider.display_for_login or state.has_account]
 
     return context
+
+
+class RecoverPasswordView(ViewSet):
+
+    """
+    **Use Cases**
+            Resets a password of a user an email address
+        **Example Requests**:
+            POST /account/recover-password
+            {"email": "staff@example.com"}
+
+        **Response Values**
+            200 or 400
+    """
+
+    def post(self, request, format=None):
+        data = request.data
+        if 'email' in data:
```

Comment:
You can just as easily do

```python
email = request.data.get('email')
if email:
    ...
```

which would also allow you to skip assigning it later.